tests/unit_tests/c_wrappers/res/enkf/test_workflow_list.py

```diff
@@ -14,37 +14,22 @@ def test_workflow_list_constructor():
 
     config_dict = {
         ConfigKeys.LOAD_WORKFLOW_JOB: [
-            {
-                ConfigKeys.NAME: "print_uber",
-                ConfigKeys.PATH: os.getcwd() + "/workflows/UBER_PRINT",
-            },
-            {
-                ConfigKeys.NAME: "HIDDEN_PRINT",
-                ConfigKeys.PATH: os.getcwd() + "/workflows/HIDDEN_PRINT",
-            },
+            [os.getcwd() + "/workflows/UBER_PRINT", "print_uber"],
```

Comment:
nitpicking of course - but would it be nicer ( and probably more performant? ) having `cwd = os.getcwd()` and then reuse the variable instead of doing all the gets? :grimacing:

Reply:
Sure. Already in. :+1: